src/content/blog/2025-08-22-multithread-linting.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "New in ESLint v9.34.0: Multithread Linting"
+teaser: "Make use of your CPU: multithreaded linting in ESLint v9.34.0 speeds up linting large projects."
+tags:
+  - Concurrency
+  - Multithreading
+  - Performance
+draft: true
+authors:
+  - fasttime
+categories:
+  - Announcements
+---
+
+## Introduction
+
+With ESLint v9.34.0, linting can now happen in true parallel. By spawning worker threads, ESLint can process multiple files at the same time, dramatically reducing lint times for large projects.  
```

Comment:
I think it would be nice to start off with how old the issue is. :smile: It's a nice story to say that an issue that was opened years ago has finally been resolved.